src/app/[locale]/projects/detail/[id]/components/Administration.tsx

```diff
@@ -19,6 +19,20 @@ const Capitalize = (text: string) => {
         ? text.split('_').reduce((s, c) => s + ' ' + (c.charAt(0) + c.substring(1).toLocaleLowerCase()), '')
         : undefined
 }
+const unescapeHtml = (str: string | undefined): string => {
+    if (!str) return ''
+
+    const htmlEntities: Record<string, string> = {
+        '&amp;': '&',
+        '&lt;': '<',
+        '&gt;': '>',
+        '&quot;': '"',
+        '&#039;': "'",
+        '&#39;': "'",
+    }
+
+    return str.replace(/&amp;|&lt;|&gt;|&quot;|&#0?39;/g, (match) => htmlEntities[match] || match)
+}
```

Comment:
Remove this, use he. 
Look on https://github.com/eclipse-sw360/sw360-frontend/pull/1299